### ISSUE TITLE
Update README.md

### ISSUE THREAD

**Issue by khalibartan** [pgmpy/pgmpy#844]
Dependencies mentioned in README is not complete list of dependencies. Also we no longer need cython so we should update that. Apart from that latest version of dependencies seems to work fine for me, so we should not make the version strict (suggestions).

**Comment by ankurankan** [comment]
@khalibartan We don't have cython in our requirements (https://github.com/pgmpy/pgmpy/blob/dev/requirements.txt). And AFAIK, I didn't update it to the latest versions because of some conflict between networkx and python versions.

**Comment by khalibartan** [comment]
@ankurankan Yes it is not in requirements.txt file but it is mentioned in our README thats what I meant.
Here is my current conda list (removed not necessary packages from the list)
```
mock                      2.0.0                    py35_0  
networkx                  1.11                     py35_0  
nose                      1.3.7                    py35_1  
numpy                     1.11.3                   py35_0  
numpydoc                  0.6.0                    py35_0  
pandas                    0.19.2              np111py35_1  
pep8                      1.7.0                    py35_0   
pgmpy                     0.1.3                     <pip>
pyparsing                 2.1.4                    py35_0  
python                    3.5.2                         0  
scipy                     0.18.1              np111py35_1  
setuptools                27.2.0                   py35_0  
six                       1.10.0                   py35_0  
wrapt                     1.10.8                   py35_0  
```
This is for python3.5. Dunno about other versions but things are working fine for me as for now.

**Comment by michaelrempel** [comment]
While you are at it can you please tell me where to find the parts that are installed. I am using a Ubuntu VM and I am not all that familiar with it. the find utility does not find dependencies.txt when I search for it after doing apt-get install. Your instructions should be clear enough for a programmer who does not understand your scripted install processes.

**Comment by ankurankan** [comment]
@michaelrempel I am sorry but I am unable to understand your exact issue. Can you please elaborate a bit please ?

**Comment by khalibartan** [comment]
@ankurankan Currently we are treating pyparsing as non-optional dependency both in code and during installation. pyparsing should be optional as only read-write modules require it.
